themes/custom/lex/css/main.css

```diff
@@ -1 +1 @@
ection:normal;-webkit-flex-direction:column;-moz-box-orient:vertical;-moz-box-direction:normal;-ms-flex-direction:column;flex-direction:column;padding:5%}}.contact-icon,.contact-text{display:inline-block}.contact-icon{font-size:45px;color:#0558A7;vertical-align:top;margin-right:20px;min-width:45px}.contact-icon i{vertical-align:top}.econdev-contact-heading{color:#0558A7;margin-bottom:25px;margin-top:20px}@media screen and (max-width: 900px){.econdev-contact-heading{text-align:center}}.econ-section:nth-child(even){background-color:#EFEFEF}.econdev-sitemast header{margin-top:15px}.econdev-sitemast #search-icon{display:none !important}.econdev-sitemast #logo img{margin:10px}.econdev-sitemast #logo .econdev-logo{margin-left:22%;font-style:italic;text-decoration:none;color:black}.econdev-sitemast .lex-region-sitemast{margin-top:15px}@media screen and (max-width: 900px){#block-econdevheaderquicklinks{display:none}#block-econdevheaderquicklinks ul{overflow:hidden}}@media screen and (max-width: 900px){.ed-help .lex-button a{padding:0.5em !important}}.econdev-img{height:50px !important}.no-scroll{overflow:hidden}.row{margin:0}ol,ul,dl{margin-bottom:0}.rounded{-webkit-border-radius:15px;-moz-border-radius:15px;border-radius:15px}.lex-region-events{background-color:#EFEFEF;display:inline-block}@media all and (-ms-high-contrast: none), (-ms-high-contrast: active){.lex-region-events{display:inline}}.lex-events-container{min-height:250px;position:relative;margin:auto}.section-padding{padding:2rem 5%}.sub-padding{padding:2rem 0}.lex-aside-list .lex-street-address,.lex-aside-list .p-locality,.lex-aside-list .lex-card-title{color:#787878}.lex-aside-list .lex-street-address:hover,.lex-aside-list .p-locality:hover,.lex-aside-list .lex-card-title:hover{color:#787878}.lex-region-sidebar #block-eventsblock,.lex-region-sidebar #block-meetingsblock{padding:2rem 0 !important}.lex-region-sidebar #block-eventsblock .lex-event-list .event-card,.lex-region-sidebar #block-meetingsblock .lex-event-list .event-card{margin-left:0 !important}.lex-region-sidebar #block-eventsblock .lex-event-list .more-link,.lex-region-sidebar #block-meetingsblock .lex-event-list .more-link{float:left;margin-right:100%}.lex-region-sidebar #block-eventsblock{margin-bottom:100%}a.lex-calltoaction:hover{background-color:#E1B111 !important}#block-urbancountycouncilnew p:nth-of-type(2){margin-right:20%;margin-left:.7rem}#block-lex-police-pagetitle{padding:2rem 0 !important}.addtoany--block svg{background-color:#787878}.flex-heading{margin-left:12px;margin-bottom:1.5em;display:block;width:100%}.flex-content{-webkit-box-orient:horizontal;-webkit-box-direction:normal;-webkit-flex-flow:row wrap;-moz-box-orient:horizontal;-moz-box-direction:normal;-ms-flex-flow:row wrap;flex-flow:row wrap;display:-webkit-box;display:-webkit-flex;display:-moz-box;display:-ms-flexbox;display:flex;-webkit-box-align:start;-webkit-align-items:flex-start;-moz-box-align:start;-ms-flex-align:start;align-items:flex-start;width:100%}
```

Comment:
So, is this CSS built by gulp? What changed here?

Reply:
I compiled the changes I made in the header.scss